datadog/dogstatsd/base.py

```diff
@@ -601,6 +600,74 @@ def disable_telemetry(self):
     def enable_telemetry(self):
         self._telemetry = True
 
+    def _parse_env_connection_overrides(self, host, port, socket_path):
+        dogstatsd_url = os.environ.get("DD_DOGSTATSD_URL")
+
+        if (
+            host == DEFAULT_HOST
+            and port == DEFAULT_PORT
+            and socket_path is None
+            and dogstatsd_url is not None
+        ):
+            parsed = urlparse(dogstatsd_url)
+            # If all values are defaults, prefer DD_DOGSTATSD_URL if present.
+            if parsed.scheme == "unix":
+                log.debug(
+                    "Found a DD_DOGSTATSD_URL matching the uds syntax, "
+                    "setting socket path %s.", dogstatsd_url
+                )
+                return host, port, dogstatsd_url
```

Comment:
Caller assigns this into `socket_path`. Should this return the full url or just the path?